docs/users-guide/Analyzing-and-Translating-Models.adoc

```diff
@@ -1478,9 +1481,35 @@ types:
 
 * `FwTlmPacketizeIdType`
 
-These types are required by the F Prime GDS, so they are included in the 
-dictionary.
-Each of these types must be an alias of an integer type.
+* `FwDpIdType`
+
+* `FwDpPriorityType`
+
+* `FwSizeStoreType`
+
+* `FwTimeBaseStoreType`
+
+* `FwTimeContextStoreType`
+
+Enumerations:
+
+* `Fw.DpState`
+* `Fw.DpCfg.ProcType`
+
+Constants:
+
+* `Fw.DpCfg.CONTAINER_USER_DATA_SIZE`
+
+The alias types and constants required by the dictionary must all be integer types.
+
+Type information for constant dictionary entries is determined by 
```

Comment:
I think this is too much information for the FPP User's Guide. It's an implementation detail about how integer types are represented in the JSON dictionary spec. Can we move this information to the JSON dictionary spec?